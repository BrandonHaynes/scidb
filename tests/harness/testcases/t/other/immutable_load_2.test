--setup

--test
--start-query-logging

create array unbounded <a:int64>[x=0:*,5,0]
--shell --store-all --command "iquery -c $IQUERY_HOST -p $IQUERY_PORT -aq "load(unbounded,'${TEST_DATA_DIR}/unbounded_empty.txt')""


 
scan(unbounded)  


--stop-query-logging
--cleanup

remove(unbounded) 
