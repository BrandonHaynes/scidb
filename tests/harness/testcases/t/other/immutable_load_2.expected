SCIDB QUERY : <create array unbounded <a:int64>[x=0:*,5,0]>
Query was executed successfully

SCIDB QUERY : <iquery -c $IQUERY_HOST -p $IQUERY_PORT -aq "load(unbounded,'${TEST_DATA_DIR}/unbounded_empty.txt')">
{x} a
{0} 1
{1} 2
{2} 3
{3} 4
{4} 5
{5} 6
{6} 7
{7} 8
{8} 9
{9} 10
{10} 11
{11} 12
{12} 13
{13} 14
{14} 15
{15} 16
{16} 17
{17} 18
{18} 19
{19} 20
{20} 21
{21} 22
{22} 23
{23} 24
{24} 25
{25} 26
{26} 27
{27} 28
{28} 29
{29} 30

SCIDB QUERY : <scan(unbounded)>
{x} a
{0} 1
{1} 2
{2} 3
{3} 4
{4} 5
{5} 6
{6} 7
{7} 8
{8} 9
{9} 10
{10} 11
{11} 12
{12} 13
{13} 14
{14} 15
{15} 16
{16} 17
{17} 18
{18} 19
{19} 20
{20} 21
{21} 22
{22} 23
{23} 24
{24} 25
{25} 26
{26} 27
{27} 28
{28} 29
{29} 30

Query was executed successfully

